### DISCUSSION TITLE
Add Support for Silero VAD

### DISCUSSION THREAD

**Discussion by fctmzold** [WyattBlue/auto-editor#540] [2. Feature Ideas]
I would like to request the integration of [Silero VAD](https://github.com/snakers4/silero-vad) into Auto-Editor. Silero VAD is a pre-trained, enterprise-grade Voice Activity Detector that offers good accuracy, fast processing models.

**Comment by timurguseynov** [comment]
Maybe that would be more accurate and wouldn't require setting the threshold

**Comment by Friebay** [comment]
Any updates on this?
